src/Chat/Chat.test.js

```diff
@@ -1,118 +1,47 @@
-//       makeGuess={
-//         async (newGuess) => {
-//           // const guess = await apiCalls.makeGuess(newGuess)
-//           // this.setState({guess})
-//           console.log('newGuess', newGuess)
-//         }
-//       }
-//       />);
-//       expect(screen.getByText('Hints')).toBeInTheDocument();
-//     })
-//     it('4. should render the `Draw Board` header', () => {
-//
-//       render(<Gamepage display={['d', '_', '_', 'o', '_', 'a', 'u', 'r']} attempts={['one', 'two', 'three']} isGenerator={null}
-//       makeGuess={
-//         async (newGuess) => {
-//           // const guess = await apiCalls.makeGuess(newGuess)
-//           // this.setState({guess})
-//           console.log('newGuess', newGuess)
-//         }
-//       }
-//       />);
-//       expect(screen.getByText('Draw Board')).toBeInTheDocument();
-//     })
-//     it('5. should render the `Attempts` header', () => {
-//
-//       render(<Gamepage display={['d', '_', '_', 'o', '_', 'a', 'u', 'r']} attempts={['one', 'two', 'three']} isGenerator={null}
-//       makeGuess={
-//         async (newGuess) => {
-//           // const guess = await apiCalls.makeGuess(newGuess)
-//           // this.setState({guess})
-//           console.log('newGuess', newGuess)
-//         }
-//       }
-//       />);
-//       expect(screen.getByText('Attempts')).toBeInTheDocument();
-//     })
-//   })
-//
-//   describe('splitDisplay()', () => {
-//     it('6. should render the `display` prop', () => {
-//
-//       render(<Gamepage display={['d', '_', '_', 'o', '_', 'a', 'u', 'r']} attempts={['one', 'two', 'three']} isGenerator={null}
-//       makeGuess={
-//         async (newGuess) => {
-//           // const guess = await apiCalls.makeGuess(newGuess)
-//           // this.setState({guess})
-//           console.log('newGuess', newGuess)
-//         }
-//       }
-//       />);
-//       for (let id = 1; id < 9; id++) {
-//         expect(screen.getByTestId(`${id}`)).toBeInTheDocument();
-//       }
-//     })
-//   })
-//
-//   describe('mapAttempts()', () => {
-//     it('6. should display each attempt', () => {
-//
-//       render(<Gamepage
-//         display={['d', '_', '_', 'o', '_', 'a', 'u', 'r']} attempts={['one', 'two', 'three']}
-//         isGenerator={null}
-//         makeGuess={
-//           async (newGuess) => {
-//           const guess = await apiCalls.makeGuess(newGuess)
-//           this.setState({guess})
-//           console.log('newGuess', newGuess)
-//           }
-//         }
-//       />);
-//       expect(screen.getByText('one')).toBeInTheDocument();
-//       expect(screen.getByText('two')).toBeInTheDocument();
-//       expect(screen.getByText('three')).toBeInTheDocument();
-//     })
-//   })
-// })
+describe('Inputs exist', () => {
+
+  describe(' `Chat` input ', () => {
+    it('1. should render the `Chat` input`', () => {
+        let mockFn = jest.fn();
```

Comment:
We might be able to give this one a bit more proper of a name. Man, I feel so British saying this.